CHANGES

```diff
@@ -8373,7 +8373,7 @@ des-cbc           3624.96k     5258.21k     5530.91k     5624.30k     5628.26k
 
   *) New OCSP utility. Allows OCSP requests to be generated or
      read. The request can be sent to a responder and the output
-     parsed, outputed or printed in text form. Not complete yet:
+     parsed, outputted or printed in text form. Not complete yet:
```

Comment:
I was surprised to see this, but it seems that Merriam-Webster agrees with it, so...